boards/shields/xenvm_dom0/boards/qemu_cortex_a53.overlay

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2023 EPAM Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/delete-node/ &sram0;
+/delete-node/ &pcie;
+/delete-node/ &uart0;
+
+/ {
+	chosen {
+		/delete-property/ zephyr,flash;
+		/delete-property/ zephyr,pcie-controller;
```

Comment:
Why are you removing those two nodes?

Reply:
Added Xen dts in `example/` directory is pretty minimalistic, so I decided to make it as simple as possible and remove all redundant configuration from original qemu_cortex_a53 device tree.

I added comment above

Reply:
Hm, I believe, so it will be better other way around: show that full QEMU machine is supported in Dom0 mode. Otherwise, there is a little value in adding shield on top of it.

Consider this point of view: you are adding a Xen shield on top of working board, but you are disabling all the devices, changing memory sizes and entry addresses (probable).  What is left of the original board? :)